pkg/antctl/raw/traceflow/command.go

```diff
@@ -88,8 +84,8 @@ func init() {
 		RunE: runE,
 	}
 
-	Command.Flags().StringVarP(&option.source, "source", "S", "", "source of the traceflow: namespace/pod or pod")
-	Command.Flags().StringVarP(&option.destination, "destination", "D", "", "destination of the traceflow: namespace/pod, pod, namespace/service, service or IP")
+	Command.Flags().StringVarP(&option.source, "source", "S", "", "source of the traceflow: namespace/Pod or Pod")
+	Command.Flags().StringVarP(&option.destination, "destination", "D", "", "destination of the traceflow: namespace/Pod, Pod, namespace/Service, Service or IP")
```

Comment:
As you are capitalizing K8s objects, namespace should apply as well.

Reply:
This time, I think I have capitalizing every single K8s objects.